jamnp/README.md

```diff
@@ -0,0 +1,90 @@
+# JAM Testnet - JAMNP Test Vectors 
+
+This describes some DRAFT **JAM Testnet** test vector files generated
+by [JAM DUNA trace data](assurances.txt), representing the majority of
+**JAMNP** request-response patterns as described in [JAM Simple
+Networking Protocol
+(JAMNP-S)](https://github.com/zdave-parity/jam-np/blob/main/simple.md).
+
+## Status
+
+This test vector set is not complete and is **NOT READY** for JAM
+implementers to build against.  It is intended to develop a course of
+action for which **JAMNP** request-response patterns and erasure
+coding vectors are necessary for a [multi-team JAM tiny
+testnet](https://github.com/jam-duna/jamtestnet/issues/69).
+
+## JAMNP Request/Response Test Vectors
+
+The table below summarizes the covered request-response types,
+providing links to the test vectors and their corresponding
+descriptions in
+[`JAMNP spec`](https://github.com/zdave-parity/jam-np/blob/main/simple.md).
+
+All data is for [tiny testnet](https://github.com/jam-duna/jamtestnet/issues/69)
+configuration with 6 validators and 2 cores.  Data is intentionally small to support initial discussion.
+
+| JAMNP Stream | Binary File | JSON File | Response Available? | Notes |
+|--------------|-------------|-----------|---------------------|-------|
+| **[UP0: Block Announcement](https://github.com/zdave-parity/jam-np/blob/main/simple.md#up-0-block-announcement)** | [BlockAnnouncement-request.bin](UP0/BlockAnnouncement-request.bin) | [BlockAnnouncement-request.json](UP0/BlockAnnouncement-request.json) | Handshake TBD |  |
+| **[CE128†: Block Request](https://github.com/zdave-parity/jam-np/blob/main/simple.md#ce-128-block-request)** | [BlockRequest-request.bin](CE128/BlockRequest-request.bin) | [BlockRequest-request.json](CE128/BlockRequest-request.json) | ✅ [BlockRequestBlocks-response](CE128/BlockRequestBlocks-response.bin) | † - Simplified as Direction=1 only |
+| **[CE131†: Ticket Distribution](https://github.com/zdave-parity/jam-np/blob/main/simple.md#ce-131132-safrole-ticket-distribution)** | [TicketDistribution-request.bin](CE131/TicketDistribution-request.bin) | [TicketDistribution-request.json](CE131/TicketDistribution-request.json) | NA | † - Simplified as "Generator" -> "all current validators" |
```

Comment:
JAMNP specify CE131 -> CE132 of "generating" validator -> "proxy" validator, then "proxy" validator -> "all current validators"

Currently simplified as CE131†  "generating" validator ->  "all current validators"